xml/System.IO/FileSystemAclExtensions.xml

```diff
@@ -110,9 +142,9 @@
         <Parameter Name="directoryInfo" Type="System.IO.DirectoryInfo" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="directoryInfo">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="directoryInfo">A directory information object that wraps the existing directory from which to obtain the security information.</param>
```

Comment:
Seems a little too wordy. `The existing directory from which to obtain the security information.` ?

Reply:
Same comment applies to the other params. Whatever we consider standard is fine.

Reply:
Sure, will change it:
```suggestion
        <param name="directoryInfo">The existing directory from which to obtain the security information.</param>
```